.github/workflows/labeler.yml

```
@@ -1,6 +1,7 @@
 name: "Pull Request Labeler"
 on:
-- pull_request_target
+  pull_request:
+    types: [opened]
 
```

Could you update like this? 😊
(Ref: https://github.com/cncf/glossary/pull/471#issuecomment-1049512552)

```suggestion
  pull_request:
    types: [opened]
    paths:
      - 'content/**'
```